src/v/kafka/server/group_manager.h

```diff
@@ -185,6 +186,8 @@ class group_manager {
 
     described_group describe_group(const model::ntp&, const kafka::group_id&);
 
+    partition_response describe_partition_producers(const model::ntp&);
```

Comment:
nit: `partition_response` is confusing as it has no context, can we create an alias for this type f.e. `partition_producers` ?